src/library/Uno.Material/MaterialLibraryResources.cs

```diff
@@ -11,6 +11,8 @@ public sealed class MaterialLibraryResources : ResourceDictionary
 		public MaterialLibraryResources()
 		{
 			// Add all ResourceDictionaries here in alphabetical order
+			this.MergedDictionaries.Add(new ResourceDictionary { Source = new Uri("ms-appx:///Uno.Material/Themes/Generic.xaml") });
```

Comment:
we don't need to add it here. theme/generic.xaml is automatically included with the nuget